### PR TITLE
commit with fixed on dashboard overview the grid box link 

### PR DESCRIPTION
**Pull request by arunkumarsekar06**
#563

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -65,4 +65,7 @@
   #===OneApps
   match '/bindservices', to: 'oneapps#bindservices', via: [:get, :post]
   match '/bindservice', to: 'oneapps#bindservice', via: [:get, :post]
+
+  #===Upload
+  match '/sign_auth', to: 'bucketkolkes#upload_signature', via: [:get]
```
Comment (indykish) [inline config/routes.rb]: rename it to a different route name. Will we use this one ?

File: app/views/buckets/_mystorage.html.erb [hunk 2]
```diff
@@ -242,4 +242,89 @@
 		}
 	});
 
+
```
Comment (indykish) [inline app/views/buckets/_mystorage.html.erb]: We may not use the java script code. So remove it.

File: Gemfile [hunk 3]
```diff
@@ -38,6 +38,7 @@ gem 'github_api', '~>0.12.4'
 gem 'gitlab', '~> 3.4.0'
 gem 'riak-client', '~> 2.2.1'
 gem 'radosgw-s3'
+gem 'jquery-fileupload-rails','~> 0.4.6' #fileupload
```
Comment (indykish) [inline Gemfile]: Move the jquery to the top (under jquery section) of the Gemfile.

File: app/assets/stylesheets/custom.css [hunk 4]
```diff
@@ -1389,7 +1389,7 @@ background-color: white;
 
 .stpack {
```
Comment (indykish) [inline app/assets/stylesheets/custom.css:1397]: @arunkumarsekar06 Is this firefox fix provided by enixel in nilavu code ? 
@megamsys/megam-dev  fyi.

**Comment by indykish** [comment]
@arunkumarsekar06  I see this pull request closed, was the fix addressed in this retrofitted then ?

**Comment by indykish** [comment]
@arunkumarsekar06   I didn't see a response to my question, hence opening it. Please don't close pull requests abruptly.

**Comment by indykish** [comment]
@arunkumarsekar06  Please fix and push again.

File: app/views/buckets/_mystorage.html.erb [hunk 5]
```diff
@@ -115,7 +112,8 @@
 </div>
 <% end %>
 <%= render :template => "users/show" %>
-
+<div id="s3">
```
Comment (thomasalrin) [inline app/views/buckets/_mystorage.html.erb]: @arunkumarsekar06 Remove this code, if you don't use

File: app/models/backup.rb [hunk 6]
```diff
@@ -20,7 +20,9 @@ class Backup < BaseFascade
   def initialize(access_key, secret_key, host)
     @client = S3::Service.new(access_key_id: "#{access_key}", secret_access_key: "#{secret_key}", host: "#{host}")
   end
-
+  def auth_sign
```
Comment (thomasalrin) [inline app/models/backup.rb]: @arunkumarsekar06 Do we use this?

File: app/controllers/bucketkolkes_controller.rb [hunk 7]
```diff
@@ -41,7 +58,9 @@ def destroy
 
   # sign our request by Base64 encoding the policy document.
   def upload_signature
```
Comment (thomasalrin) [inline app/controllers/bucketkolkes_controller.rb]: @arunkumarsekar06 If we don't use, delete it.

File: app/views/bucketkolkes/_upload.html.erb [hunk 8]
```diff
@@ -0,0 +1,23 @@
+<div class="modal-dialog" style="max-width:400px">
```
Comment (thomasalrin) [inline app/views/bucketkolkes/_upload.html.erb]: @arunkumarsekar06 Remove if we don't use

**Pull request merged**